src/workflows/stats/bidsResults.m

```diff
@@ -121,7 +121,12 @@
 
   opt.dir.output = opt.dir.stats;
 
-  [~, opt] = setUpWorkflow(opt, 'computing GLM results');
+  [BIDS, opt] = setUpWorkflow(opt, 'computing GLM results');
+
+  if isempty(opt.model.file)
```

Comment:
That's where the  main error was coming from. 

I had not thought that it would be possible to run Results and RFX with the default model. So now that should be possible.